lib/setupFirebase.js

```diff
@@ -1,18 +1,24 @@
 require('dotenv').load();
-var admin = require('firebase-admin');
+const admin = require('firebase-admin');
 
 const testing = process.env.NODE_ENV !== 'production';
-const key = testing ? process.env.TESTING_FIREBASE_TOKEN : process.env.FIREBASE_TOKEN
+const key = testing ? process.env.TESTING_FIREBASE_TOKEN : process.env.CLOUD_FIREBASE_TOKEN
 var firebasekey = key.replace(/\\n/g, '\n');
 console.log("TESTING:", testing)
-admin.initializeApp({
-  credential: admin.credential.cert({
-    projectId: testing ? process.env.TESTING_PROJECT_ID : process.env.FIREBASE_ID,
-    clientEmail: testing ? process.env.TESTING_CLIENT_EMAIL : process.env.FIREBASE_EMAIL,
-    privateKey: firebasekey,
-  }),
-  databaseURL: testing ? process.env.TESTING_DATABASE_URL : process.env.FIREBASE_DB_URL,
-});
 
-// admin.database.enableLogging(true);
-module.exports = admin.database();
+var app = admin.initializeApp(
+    {
+        credential: admin.credential.cert({
+            projectId: process.env.FIREBASE_ID,
+            clientEmail: process.env.FIREBASE_EMAIL,
+            privateKey: firebasekey
+        }),
+        databaseURL: process.env.FIREBASE_DB_URL
+    }
+);
+
+// Export together as single object
+module.exports = {
+    firestore: app.firestore(),
+    realtimedb: app.database()
```

Comment:
In future scripts, you will have to specify your target for where to point the data:
I.e.: `firebasedb.firestore` or `firebasedb.realtimedb`